lms/djangoapps/courseware/tests/test_submitting_problems.py

```diff
@@ -473,17 +475,11 @@ def test_grade_with_max_score_cache(self):
                 module_state_key=self.problem_location('p2')
             ).exists()
         )
-        location_to_cache = unicode(self.problem_location('p2'))
-        max_scores_cache = grades.MaxScoresCache.create_for_course(self.course)
 
-        # problem isn't in the cache
-        max_scores_cache.fetch_from_remote([location_to_cache])
-        self.assertIsNone(max_scores_cache.get(location_to_cache))
```

Comment:
Additionally, if you want to check that the `max_score` is stored in the cached data, you can look for it in the structure returned by `BlockStructureCache.get`.  But I agree that that checking should be done by the `GradesTransformer` unit tests instead.